.github/workflows/ut.yaml

```diff
@@ -46,11 +46,21 @@ jobs:
           ./build.sh -l
       - name: Run Unittest
         run: |
-          ./build.sh -t Release -u
-          if [[ -f "output/unittest/faiss_test" ]]; then
-            ./output/unittest/faiss_test
+          if [[ ${{ matrix.os }} == 'centos7' || ${{ matrix.os }} == 'macos-11' ]]
```

Comment:
can you write in this way:

if [[ ${{ matrix.os }} == 'ubuntu-18.04' ]]
then
  $disk_ann_opt = -d
fi

./build.sh ${disk_ann_opt} -t Release -u
if [[ -f "output/unittest/faiss_test" ]]; then
  ./output/unittest/faiss_test
fi
  ./output/unittest/test_knowhere
fi